README.md

```diff
@@ -1,58 +1,95 @@
-# erlcloud: Cloud Computing APIs For Erlang #
+# erlcloud: AWS APIs library for Erlang #
 
 [![Build Status](https://secure.travis-ci.org/erlcloud/erlcloud.png?branch=master)](http://travis-ci.org/erlcloud/erlcloud)
 
-Service APIs implemented:
+This library is not developed or maintained by AWS thus lots of functionality is still missing comparing to [aws-cli](https://aws.amazon.com/cli/) or [boto](https://github.com/boto/boto).
+Required functionality is being added upon request.
 
+Service APIs implemented:
 - Amazon Elastic Compute Cloud (EC2)
 - Amazon Simple Storage Service (S3)
 - Amazon Simple Queue Service (SQS)
-- Amazon SimpleDB
-- Amazon Mechanical Turk
-- Amazon CloudWatch (MON)
-- Amazon DynamoDB (ddb2) 
+- Amazon DynamoDB & DDB streams (ddb2)
+- Amazon Autoscalling(as)
+- Amazon CloudTrail (CT)
+- Cloud Formation (CFN)
+- ElasticLoadBalancing (ELB)
+- Identity and Access Management (IAM)
+- Kinesis
+- CloudWatch
+- MechanicalTurk
+- Simple DB (SDB)
+- Relational Data Service (RDS)
+- Simple Email Service (SES)
+- Short Token Service (STS)
+- Simple Notification Service (SNS)
+- Web Application Firewall (WAF)
+- and more to come
+
+Majority of API functions have been implemented.
+Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.
+Please send issues and patches.
+
+The libraries can be used two ways:
+- either you can specify configuration parameters in the process dictionary. Useful for simple tasks
+- you can create a configuration object and pass that to each request as the final parameter. Useful for Cross AWS Account access
 
-All API functions have been implemented.  Not all functions have been thoroughly tested, so exercise care when integrating this library into production code.  Please send bug reports and patches.
+## Roadmap ##
 
-The libraries can be used two ways: either you can specify configuration parameters in the process dictionary, or you can create a configuration object and pass that to each request as the final parameter.
+Below is the proposed library roadmap update along with regular features and fixes.
 
-## Getting started ##
-You need to clone the repository and download rebar (if it's not already available in your path).
+- v0.13.9
+ * Existing code
 
-```
-git clone https://github.com/erlcloud/erlcloud.git
-cd erlcloud
-wget http://cloud.github.com/downloads/basho/rebar/rebar && chmod u+x rebar
-```
+- v2.0.X
```

Comment:
the reason is we have v1.X.X in AL fork.
I was planning to import laster AL tag into here and even publish it to HEX once.